### PR TITLE
README - Add branch flow

### PR DESCRIPTION
**Pull request by jomey**
Move the definitions from the sample project repository to here and centralize
explanations and practice instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,10 +210,96 @@ Here the steps:
...
+    ```shell
+    main
+    * feature_xyz
+    ```
+
+1. Make changes within the new branch
+
+   With this practice repository, you have to create a folder within
+   **`contributions`** folder with your github username as the folder name. Use
+   a new markdwon file (`.md`) or text file (`.txt`) file to practice git with.
```
Comment (lsetiawan) [inline README.md:264]: I'm wondering if we also should allow `.rst`? Some people might be familiar with that.

File: README.md [hunk 2]
```diff
@@ -182,10 +210,96 @@ Here the steps:
...
+   ![pull request](images/pull-request-branch.png)
+
+1. Go to the pull request tab and celebrate!
+
+   ```shell
+    https://github.com/snowex-hackweek/learning-git/pulls
+    ```
+   
+1. Once your Pull Request is merged, clean up and delete the branch
+   locally and on the website.
```
Comment (lsetiawan) [inline README.md]: Will this be covered anywhere in the tutorial; the local and remote clean up?

**Review by lsetiawan — APPROVED** [review]
This is really great @jomey! I really like the flow and your instructions are very clear and concise. I have a few questions in the comments above. 

Another question I have is, will you be giving push access to the participants to this repo to create new branch? Seems like that's needed if they were gonna try to create branch within the repo itself. I got my admin power access back! so let me know if you need this set up 😄

File: README.md (hunk 1 above)
Comment (jomey) [reply README.md:264]: I think the two options should cover most of participants. It is also a simple practice run and I like simple options :-)

File: README.md (hunk 2 above)
Comment (jomey) [reply README.md]: Good point. I can add another section regards that.

**Comment by jomey** [comment]
> Another question I have is, will you be giving push access to the participants to this repo to create new branch? Seems like that's needed if they were gonna try to create branch within the repo itself. I got my admin power access back! so let me know if you need this set up smile

I have given the participants and instructors `write` permissions to this repo. That should take care if it (right?)

**Comment by lsetiawan** [comment]
> I have given the participants and instructors write permissions to this repo. That should take care if it (right?)

Yepp! That should be good 😸

**Comment by jomey** [comment]
@lsetiawan - I will open a PR for the upstream too

**Comment by lsetiawan** [comment]
> I will open a PR for the upstream too

You the best! 🥳

**Pull request merged**